model/clusters_mgmt/v1/add_on_installation_operator_type.model

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Representation of an add-on installation operator
+class AddOnInstallationOperator {
```

Comment:
Does this represent an entity with its own endpoint in the API? If not then it should be a `struct` instead of a `class`.

Reply:
good catch, thanks :+1: